ISISNeutronMuon/MDANSE
# Remove or replace several Cython extensions

**Description of work**
Two Cython extensions have been identified as unused and removed. A third one mt_fast_calc, has been replaced with a faster numpy equivalent.

**Fixes**
1. Deleted sd_fast_calc.pyx
2. Deleted change_coordinates.pyx
3. Deleted mt_fast_calc.pyx and replaced the corresponding function call with equivalent Python code using numpy.

**To test**
Please run MolecularTrace analysis on a trajectory using the protos branch. Switch to this branch and run the same MolecularTrace again. The results should be the same.
